test/e2e/kubevirt.go

```diff
@@ -1095,15 +1110,33 @@ passwd:
 		}
 
 		iperfServerScript = `
-dnf install -y psmisc procps
+#!/bin/bash -xe
 iface=$(ifconfig  |grep flags |grep -v "eth0\|lo" | sed "s/: .*//")
+iface=${iface:-eth0}
+
 ipv4=$(ifconfig $iface | grep "inet "|awk '{print $2}'| sed "s#/.*##")
-ipv6=$(ifconfig $iface | grep inet6 |grep -v fe80 |awk '{print $2}'| sed "s#/.*##")
 if [ "$ipv4" != "" ]; then
 	iperf3 -s -D --bind $ipv4 --logfile /tmp/test_${ipv4}_iperf3.log
+	sleep 1
```

Comment:
:sweat_smile: why do wee need all the sleeps?

Reply:
To detect stuff like binding errors, iperf3 -D do not fail if it cannot bind the socket.